bam_sort.c

```diff
@@ -1506,9 +1506,7 @@ int bam_merge(int argc, char *argv[])
 
     hts_srand48(random_seed);
     if (!(flag & MERGE_FORCE) && strcmp(fnout, "-") != 0) {
-        FILE *fp = fopen(fnout, "rb");
-        if (fp != NULL) {
-            fclose(fp);
+        if (access(argv[optind], F_OK) == 0) {
```

Comment:
`fnout`!!!!

Reply:
Ugh!  That was dumb.   I think this is probably a cut/paste error when I redid the change.  It wouldn't let me pull from develop to update it due to a conflict, so I reset it pulled, and then hacked in the one/two lines of code again.   Wrongly ;)